survey/survey_app/models.py

```diff
@@ -24,11 +30,18 @@ class Option(models.Model):
     def __str__(self):
         return self.text
 
+class AnswerManager(models.Manager):
```

Comment:
Why do we need this class @newDaria ?
Is there another solution for counting the user submission per day?

Reply:
The AnswerManager class is created to add a custom method user_submission_count_for_day, which allows us to get the count of answers submitted by a specific user on a given day. By having this custom manager, we can encapsulate the logic for this specific query within the model itself. This improves code organization and keeps the logic related to counting user submissions within the model.

Reply:
Another possible option:

def user_submission_count_for_day(user, date):
    return Answer.objects.filter(creator=user, created_at__date=date).count()